raiden/network/transport/matrix.py

```diff
@@ -743,15 +775,7 @@ def _update_address_presence(self, address):
 
         if new_state == self._address_to_presence.get(address):
             return
-        self.log.debug(
```

Comment:
IMO that log is rather useful.

Reply:
It's mostly covered by the log above, `Changing user presence state`, so I removed it because it was spamming the logs with mostly the same info.

Reply:
But this is the actual important one since the address presence is what determines the network reachability. So I'd say remove the above on and keep this.